webcam-capture-drivers/driver-raspistill/README.md

```diff
@@ -0,0 +1,204 @@
+
+Image parameter commands
+
+-sh, --sharpness	: Set image sharpness (-100 to 100)
+-co, --contrast	: Set image contrast (-100 to 100)
+-br, --brightness	: Set image brightness (0 to 100)
+-sa, --saturation	: Set image saturation (-100 to 100)
+-ISO, --ISO	: Set capture ISO
+-vs, --vstab	: Turn on video stabilisation
+-ev, --ev	: Set EV compensation - steps of 1/6 stop
+-ex, --exposure	: Set exposure mode (see Notes)
+-fli, --flicker	: Set flicker avoid mode (see Notes)
+-awb, --awb	: Set AWB mode (see Notes)
+-ifx, --imxfx	: Set image effect (see Notes)
+-cfx, --colfx	: Set colour effect (U:V)
+-mm, --metering	: Set metering mode (see Notes)
+-rot, --rotation	: Set image rotation (0-359)
+-hf, --hflip	: Set horizontal flip
+-vf, --vflip	: Set vertical flip
+-roi, --roi	: Set region of interest (x,y,w,d as normalised coordinates [0.0-1.0])
+-ss, --shutter	: Set shutter speed in microseconds
+-awbg, --awbgains	: Set AWB gains - AWB mode must be off
+-drc, --drc	: Set DRC Level (see Notes)
+-st, --stats	: Force recomputation of statistics on stills capture pass
+-a, --annotate	: Enable/Set annotate flags or text
+-3d, --stereo	: Select stereoscopic mode
+-dec, --decimate	: Half width/height of stereo image
+-3dswap, --3dswap	: Swap camera order for stereoscopic
+-ae, --annotateex	: Set extra annotation parameters (text size, text colour(hex YUV), bg colour(hex YUV))
+-ag, --analoggain	: Set the analog gain (floating point)
+-dg, --digitalgain	: Set the digital gain (floating point)
+
+
+Notes
+
+Exposure mode options :
+off,auto,night,nightpreview,backlight,spotlight,sports,snow,beach,verylong,fixedfps,antishake,fireworks
+
+Flicker avoid mode options :
+off,auto,50hz,60hz
+
+AWB mode options :
+off,auto,sun,cloud,shade,tungsten,fluorescent,incandescent,flash,horizon
+
+Image Effect mode options :
+none,negative,solarise,sketch,denoise,emboss,oilpaint,hatch,gpen,pastel,watercolour,film,blur,saturation,colourswap,washedout,posterise,colourpoint,colourbalance,cartoon
+
+Metering Mode options :
+average,spot,backlit,matrix
+
+Dynamic Range Compression (DRC) options :
+off,low,med,high
+
+Preview parameter commands
+
+-gs, --glscene	: GL scene square,teapot,mirror,yuv,sobel,vcsm_square
+-gw, --glwin	: GL window settings <'x,y,w,h'>
+```
+## Driver Parameters
+this driver makes above arguments configurable, when driver instance created, it will load arguments step by step as follows,
+1. load built-in parameters in defaults.properties packed in jar. one line one argument
+2. search raspistill.properties in classpath root, if exists, will merge to default.properties.
+3. search system properties which starts with raspistill. then merge to properties
+4. RaspistillDriver class is builder design pattern, you can change configurations when create new driver instance, code as *new RaspistillDriver().width(600).height(400)....*
+
+## Capture Driver License
+
+Copyright (C) 2012 - 2019 Bartosz Firyn <https://github.com/sarxos> and contributors
```

Comment:
You should put your name here @alexmao86 since you are the author of this capture driver :)